javascript-functions/main.js

```diff
@@ -0,0 +1,39 @@
+function convertMinutesToSeconds(minutes) {
+  var seconds = minutes * 60;
+  return seconds;
+}
+var convertMinutesToSecondsResult = convertMinutesToSeconds(8);
+console.log('value of convertMinutesToSecondsResult:', convertMinutesToSecondsResult);
+
+function greet(name) {
+  var greeting = 'Hey' + ' ' + name;
+  return greeting;
+}
+var fullGreeting = greet('Brandon');
+console.log('value of fullGreeting:', fullGreeting);
+
+function getArea(width, height) {
+  var area = width * height;
+  return area;
+}
+var areaResult = getArea(17, 42);
+console.log('value of areaResult:', areaResult);
+
+function getFirstName(person) {
+  var firstName = person.firstName;
+  return firstName;
+}
+var person = {
+  firstName: 'Lelouche',
+  lastName: 'Lamperouge'
+};
+getFirstName({ firstName: 'Lelouche', lastName: 'Lamperouge' });
+console.log('value of getFirstName:', person.firstName);
+
+function getLastElement(array) {
+  var lastElement = array.length - 1;
+  return lastElement;
```

Comment:
here you want to return the last element of the array, not the number of the last index.